test/IFrame.spec.ts

```diff
@@ -20,5 +20,32 @@ describe('Test IFrames', () => {
                 });
             });
         });
+
+        describe('When the user is at the main context', () => {
+            it('Then, the title should be "Sample Iframe page"', async () => {
+                await expect(iFramePage.getMainPageTitle()).toBe('Sample Iframe page');
+            });
+        });
+
+        describe('When the user switches to the IFrame 1 context', () => {
+            beforeAll(async () => {
+                await iFramePage.formIFrameContext();
+            });
+
+            it('Then, the title should be "Practice Automation Form"', async () => {
+                await expect(iFramePage.getFormIFramePageTitle()).toBe('Practice Automation Form');
+            });
+        });
+
+        describe('When the user switches to the main context', () => {
+            beforeAll(async () => {
+                await iFramePage.mainContext;
```

Comment:
I don`t know this line, here you don`t execute the function. 

I think that the correct way is: 
`await iFramePage.mainContext()`

Please tell me if I'm right